frontend/src/components/atoms/Shared/controls/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Button as MuiButton } from "@material-ui/core";
+
+export default function Button(props: any) {
+  const { text, size, color, variant, onClick, ...other } = props;
+  return (
+    <MuiButton
+      variant={variant || "contained"}
+      size={size || "large"}
+      color={color || "primary"}
+      onClick={onClick}
+      {...other}
+    >
+      {text}
+    </MuiButton>
+  );
+}
```

Comment:
To bym wywalił ;)
Ale technicznie kod dobry, ale chyba nie warto tego umieszczać.
Chyba że ustawiłbyś wartości na twardo 
- czyli np nazwa 
`PrimaryMuiButton`
- i wartości na stałe (ze zmiennym onClick oczywiście) - to by miało sens

(jeszcze przeczytam resztę kodu)

Reply:
```variant={variant || "contained"}
      size={size || "large"}```
      
Czekaj, bo to są typy, zgadza się?

Reply:
Raczej buttona wyrzucić bezpośrednio do "atoms"
Czyli " frontend/src/components/atoms/Button.tsx "
Albo może " frontend/src/components/atoms/controls/Button.tsx "
Niepotrzebnie ten folder "Shared" jest - zapomniełam go usunąć w moim PR.
Usuń proszę :)

Reply:
Bo tu chyba @nowakprojects będzie miał podobne zdanie, że nie powinniśmy przepisywać
i że albo usunąć, albo na mu stałe wartości i to miałoby sens

Reply:
Bo w tej chwili przycisk i tak musimy ustawiać. Jeśli natomiast damy tu stałe to musimy tylko podać callback funkcję oraz tekst i będzie mniej pracy.
A zmiana stylu nastąpi automatycznie we wszystkich przyciskach, jak zmienimy tylko tu wartości.
No i nie będziemy podawać powtarzających się parametrów jak 
```
variant={variant || "contained"}
      size={size || "large"}
      color={color || "primary"}
```
      
      I to miałoby sens.
      
      Ale ogólnie super, że takie rzeczy programujesz, fajnie Tobie idzie.
      Ale i tak do poprawy

Reply:
@nowakprojects zerknij proszę tez

Reply:
no też się zastanawiałem, czy ten input trochę nie na siłę - dobrze wiedzieć ... będę ogarniał jutro :-)
Dzięki wielkie za komentarze!

Reply:
Jak dla mnie totalnie nie ma sensu robić tutaj osobnego komponentu. 

Mamy Button z MaterialUI. Teraz się ograniczamy, bo wybierasz tylko pod grupę możliwości Buttona z MaterialUI. I cały nasz design jest związany z MaterialUIem, mogłoby to się obronić, gdybyś chciał nas odciąć od jakiejś zewnętrznej biblioteki i wybrać jakieś funkcjonalności, które nas interesują, ale tutaj tego nie widzę.
Skupcie się na feature, a nie wyciągani takich małych części :) Możemy pójść w duplikację, dopiero po czasie okażę się, czy potrzeba jest coś wyciągnąć. YAGNI.

Dodatkowo nie określasz typów propsów.

Reply:
okej, to wracam z tymi zmianami :-)